### ISSUE TITLE
UI ignores anti-aliasing setting of client machine (Windows 7)

### ISSUE THREAD

**Issue by germanicianus** [FAForever/downlords-faf-client#801]
Despite anti-aliasing is disabled on the client machine, the UI has it enabled. Compare the following two cutouts of one screenshot. It's already visible at original size and cannot be overlooked when enlarged to 200 %.

Some Chinese characters (complete translation not available yet) are anti-aliased, some are not which makes the anti-aliased characters unclear the label lines wiggly. This may be a font issue where some characters are taken from the primary font, but others are not available and are taken from a substitute font.

![font-anti-aliasing_comparison](https://user-images.githubusercontent.com/37223176/37546917-56ca72ee-296f-11e8-9c5f-2b36e769621d.png)

**Comment by micheljung** [comment]
Not that I'm an expert in this, but JavaFX doesn't use the OS' font rendering (after all, it's platform independent and aims to look the same on all OS').

See also: [How to force anti-aliasing in JavaFX fonts](https://stackoverflow.com/a/24419312/513451) and [JavaFX Font Rendering Bug & Workaround (Linux)](http://elouh.com/?p=23)

**Comment by germanicianus** [comment]
I'm coming from Swing, so I am neither an expert in this :) . It's just a minor glitch which we should track that it doesn't get lost. In the end it's currently just a bit strange to the eye for Asian characters. Maybe we find some time to have a look at it :) .

Swing respects the OS' settings - shot on my machine:
![swing_dialog](https://user-images.githubusercontent.com/37223176/37549799-f5c56d90-2983-11e8-9ddf-92d6c862f8ba.png)

**Comment by germanicianus** [comment]
So JavaFX' goal is different from the Swing goal which tries to mimic the OS Look'n'Feel as well as possible.

I've tried the following options and also searched the net, but I only was able to change the anti-aliasing, but not to disable it completely. At least the colored (sub-pixel) anti-aliasing can be changed to gray-anti-aliasing.
```
-Dprism.lcdtext=false
-Dprism.subpixeltext=false
-Dprism.text=t2k
```
But I've also found an interesting article titled [JavaFx 8 command line options](http://werner.yellowcouch.org/log/javafx-8-command-line-options/).
